gulpfile.js

```diff
@@ -24,16 +24,21 @@ var argv = yargs
   .command('build', 'generate optimimized third party library for production',
     function(yargs) {
       argv = yargs
-        .usage('Usage: $0 build [--minify]')
+        .usage('Usage: $0 build [--minify] [--output_directory]')
         .option('minify', {
           describe: 'Whether to minify third-party dependencies'
-        }).argv;
+        })
+        .option('output_directory', {
+          describe: 'A path to the directory where the files will be generated'
+        })
+        .argv;
     })
   .command('start_devserver', 'start GAE development server',
     function(yargs) {
       argv = yargs
         .usage('Usage: $0 start_devserver [--gae_devserver_path]' +
-         '[--clear_datastore] [--enable_sendmail] [--use_minification]')
+         '[--clear_datastore] [--enable_sendmail] [--use_minification]' +
+         '[--prod_env]')
```

Comment:
sorry what the difference between these flags `--use_minification` and `--prod_env`?  because can you run in prod_env without minification? I suggest remove use_minification if they do not have different functionality.